traits/has_traits.py

```diff
@@ -258,7 +258,7 @@ def _get_instance_handlers ( class_dict, bases ):
                         arg_lists.append( arg_list )
 
     # Merge in the information from the class dictionary:
-    for name, value in class_dict.items():
```

Comment:
This change could probably be reverted (I may have accidentally introduced it during the merge).

Reply:
Done.